### DISCUSSION TITLE
Does this repository support logging val_loss?

### DISCUSSION THREAD

**Discussion by Pevernow** [bghira/SimpleTuner#1310] [Q&A]
I am using the original Sana official repository for training, which only provides a loss curve. Considering that it does not split the validation set, this should be just train_loss.

This curve seems to be difficult to present the true situation of the model during training. Specifically, I trained on 50,000 images for about 10 epochs, and the loss curve did not drop significantly.

I have tried the official recommended learning rate of 1e-4 and the recommended learning rate of 1e-5 in this repository. The situation is even worse at 1e-4, and the loss even keeps rising.

Due to the above situation, it is difficult for me to evaluate the training effect of the model. There is no obvious change in the pictures generated by the verification. I urgently need val_loss for evaluation.

(I am just a beginner and am not familiar with model training)
![{D4C3E704-8EA9-4E8A-B108-4ACF8C4F3C30}](https://github.com/user-attachments/assets/8af64edc-2a14-46a7-8848-fd6fdd57c33c)

**Comment by Pevernow** [comment]
If @bghira  or anyone else would be willing to answer, I would be grateful.

**Comment by bghira** [comment]
#1304

**Comment by Pevernow** [comment]
Thanks
